### PR TITLE
Added Home Assistant (0.60)

### PR DESCRIPTION
**Pull request by chriskacerguis**
Added Home Assistant 0.60 (and also .DS_Store to the .gitignore).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .*.swp
 *~
+.DS_Store
```
Comment (rawmind0) [inline .gitignore]: Please, remove this file from the PR.

File: templates/home-assistant/0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  home-assistant:
+    image: homeassistant/home-assistant:0.60
+    privileged: true
+    stdin_open: true
+    network_mode: host
+    volumes:
+    - /opt/homeassistant:/config
```
Comment (rawmind0) [inline templates/home-assistant/0/docker-compose.yml]: Please, use the rancher-compose variable instead hardcoded path.
```
-    - /opt/homeassistant:/config
+    - ${config_path}:/config
```

File: templates/home-assistant/0/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  home-assistant:
+    image: homeassistant/home-assistant:0.60
+    privileged: true
```
Comment (rawmind0) [inline templates/home-assistant/0/docker-compose.yml]: Please, use the rancher-compose variable instead hardcoded path.
```
-    privileged: true
+    privileged: ${privileged}
```

**Comment by chriskacerguis** [comment]
wow...sorry.  Pro Tip: Don't do a PR when you are sleep deprived.

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,2 @@
 .*.swp
-*~
-.DS_Store
\ No newline at end of file
+*~
```
Comment (rawmind0) [inline .gitignore]: Please, remove whole file `.gitignore` from the PR, not just a line.

**Comment by chriskacerguis** [comment]
Closing.  This repo's PR review process cannot keep up with the changes from most open source projects (not a dig, totally understandable)

**Comment by rawmind0** [comment]
Really?? Waiting for your action, remove a file from the PR, since 25 days ago.

**Comment by chriskacerguis** [comment]
@rawmind0 This was a new PR...the previous one:

https://github.com/rancher/community-catalog/pull/604 was opened on Aug 25, 2017.  It didn't get looked at until Dec 14, 2017

Respectfully, in the Aug 25 - Dec 14 date there were numerous Home Assistant releases...there are essentially releases every 2 weeks.  It would seem to be that there are other priorities (which, again is totally fine).  It's just easier to maintain a separate catalog that I can merge / review myself.

To your point about the removing a file, TBH after the #604 PR sat with no response for months, this one become less of a priority (and thus I just ignored it).

Again, I meant no disrespect from my comment.  :)

**Comment by rawmind0** [comment]
Previous one, may be you are right..Sorry about it.. But not with this one.. it's a shame not to merge it finally.. ;)

**Pull request closed without merging**